relay_handler.js

```diff
@@ -498,6 +663,144 @@ function logForwardedError(errFrame) {
     self.logError(err, errors.classify(err) || 'UnexpectedError');
 };
 
+LazyRelayOutReq.prototype._observeCallResFrame =
+function _observeCallResFrame(frame, now) {
+    var self = this;
+
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.inbound.response.size',
+        'counter',
+        frame.size,
+        new stat.InboundResponseSizeTags(
+            self.inreq.callerName,
+            self.inreq.serviceName,
+            self.inreq.endpoint
+        )
+    ));
+
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.outbound.response.size',
+        'counter',
+        frame.size,
+        new stat.OutboundResponseSizeTags(
+            self.inreq.serviceName,
+            self.inreq.callerName,
+            self.inreq.endpoint
+        )
+    ));
+
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.inbound.calls.latency',
+        'timing',
+        now - self.inreq.start,
```

Comment:
This is once more cheating. Need to do a `Date.now()` here to include `socket.write()` time.